### PR TITLE
Iv 4282 enhance upgrade script

### PR DESCRIPTION
**Pull request by psschroeter**
Few things:
- Finished out nick's work to add retry/timeout to all rsc calls. Some of this was already on the branch when I inherited it, just finished it out for consistency sake.
-  Moved UPGRADES_LOCATION parameter from rll/upgrade.sh to rll/setup-automatic-upgrade.sh and from rlw/upgrade.sh to rlw/setup-automatic-upgrade.sh. Added a new parameter UPGRADE_VERSION to rll/upgrade.sh and rlw/upgrade.sh to accept a simple text version to upgrade to, i.e. '10.5.4'. This conceptually simplified things a bit as setup-automatic-upgrade is now like a a subscription service.

### CODE REVIEW

File: rlw/setup-automatic-upgrade.ps1 [hunk 1]
```diff
@@ -19,37 +27,68 @@
...
     Write-Output 'Automatic upgrade never enabled - no action done'
   }
 } else {
-  # Random hour 0-23
   $jobHour = Get-Random -Min 0 -Max 24
-  # Random minute 0-59
   $jobMinute = Get-Random -Min 0 -Max 60
   # Create time format of ##:## needed for SCHTASKS
   $jobStartTime = '{0:d2}:{1:d2}' -f $jobHour, $jobMinute
+  Set-Content $execScript "
```
Comment (douglaswth) [inline rlw/setup-automatic-upgrade.ps1]: Would it be better to quote this as a PowerShell here-string?

``` powershell
Set-Conent $execScript @"
# ...
"@
```

Comment (psschroeter) [reply rlw/setup-automatic-upgrade.ps1]: Yes will fix

Comment (psschroeter) [reply rlw/setup-automatic-upgrade.ps1]: Alright done

**Review by douglaswth — APPROVED** [review]
LGTM

**Pull request merged**